.gitmodules

```diff
@@ -46,3 +46,6 @@
 [submodule "third_party/c-dt"]
```

Comment:
Please, follow our style guide regarding commit messages.

https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message

At least closing keywords, but also some description of the problem you solved here. Notable implementation details as well: say, things, where you made a deliberate choice between different ways. (In sum, a commit message is presentation of your work for one who likely don't know all details when start reading.)

Please, don't miss a [changelog entry](https://github.com/tarantool/tarantool/blob/master/doc/changelogs.md). Look over examples in the `changelogs/unreleased` directory.

If appropriate (if we'll actually change our dependencies) don't miss a [documentation request](https://github.com/tarantool/docbot) (you can just see example in other commit messages).

Reply:
Added file `changelogs/unreleased/add-http2-support-for-http-client` with content:

```
## feature/build

 * Added bundling of libnghttp2 for bundled libcurl to support http2
   for http client.
```

Reply:
I'll comment in place:

> build: link bundled libcurl with nghttp2
>
> Before this patch Tarantool http client did not support HTTP2,

Nit: It is usually written HTTP/2.

> HTTP2 is needed to support H20 HTTP server. This patch

There is nothing specific for h2o server. If you want reasons to enable HTTP/2 support in our client: efficiency offered by the protocol and potential ability to interact with a GRPC server.

> introduces bundling libnghttp2 as an implementation of HTTP2 for libcurl.

Reply:
New commit message:

```
    build: link bundled libcurl with nghttp2
    
    Before this patch Tarantool http client did not support HTTP/2.
    The reasons to enable HTTP/2 support are efficiency offered by
    the protocol and potential ability to interact with a GRPC server.
    The CMake version requirement is updated from 3.2 to 3.3, because
    we need generator expressions in cmake for nghttp2 support.
    
    Closes #5771

```

Reply:
And small updates in CHANGELOG:

```diff
## feature/build

* Added bundling of libnghttp2 for bundled libcurl to support http2
- for http client.
+ for http client. The CMake version requirement is updated from 3.2 to 3.3.
```

Reply:
Okay, thanks.